tests/e2e/configurer/config/constants.go

```diff
@@ -13,4 +13,8 @@ const (
 	PropBufferBlocks float32 = 5
 	// max retries for json unmarshalling
 	MaxRetries = 60
+	// Minimum deposit value for a proposal to enter a voting period.
+	MinDepositValue = 500000000
```

Comment:
```suggestion
	MinDepositValue = 10000000
```

If the min deposit param defaults to 10000000 like before this needs to get reduced, otherwise every time the `InitialMinDeposit` is actually depositing >100% so it wont properly test that the 25% prop deposit requirement is actually working or not

Reply:
I don't understand - `MinDepositValue` is the minimum deposit necessary for the proposal to move to the voting period so this is expected. For determining an acceptable value for the initial deposit, we divide this by 4

Reply:
https://github.com/osmosis-labs/osmosis/blob/31b524cadb16f0f79ff13ae5020716eff3e5cbd3/tests/e2e/configurer/config/constants.go#L19

Reply:
Right what I am saying is I think the minimum deposit requirement when starting a chain up defaults to 10000000, not 500000000 (what mainnet is)

Reply:
So you are hardcoding 500000000 here but it is still utilizing 10000000 as the min deposit param

Reply:
Yeah I just checked and that is the case

```
czar@Adams-MacBook-Pro osmosis % docker exec -it 017fdf84fbdc bash
root@017fdf84fbdc:~# osmosisd q gov params
deposit_params:
  max_deposit_period: "172800000000000"
  min_deposit:
  - amount: "10000000"
    denom: uosmo
tally_params:
  quorum: "0.334000000000000000"
  threshold: "0.500000000000000000"
  veto_threshold: "0.334000000000000000"
voting_params:
  voting_period: "19000000000"
```

Reply:
This is now addressed